bgpd/bgp_nht.c

```diff
@@ -1515,6 +1516,10 @@ void bgp_nht_reg_enhe_cap_intfs(struct peer *peer)
 		return;
 
 	bgp = peer->bgp;
+
+	/* Shouldn't enable RA if they are disabled */
+	assert(!CHECK_FLAG(bgp->flags, BGP_FLAG_IPV6_NO_AUTO_RA));
```

Comment:
I doubt on this assert, why is this needed? Can't be handle it more gracefully?

Reply:
This function is called from only one place that is already covered by checking the flag, so I thought about making an assert there. Now I've changed it to a regular check with early return.